pkg/controller/replicaset/replica_set_test.go

```diff
@@ -1218,10 +1218,11 @@ func TestExpectationsOnRecreate(t *testing.T) {
 		t.Fatal(err)
 	}
 
-	err = wait.PollImmediate(100*time.Millisecond, informerSyncTimeout, func() (bool, error) {
-		logger.V(8).Info("Waiting for queue to have 1 item", "length", manager.queue.Len())
-		return manager.queue.Len() == 1, nil
-	})
+	err = wait.PollUntilContextTimeout(context.Background(), 100*time.Millisecond, informerSyncTimeout, true,
```

Comment:
the same here and probably in all tests below.

Reply:
get it